addons/mail/static/src/core/web/message_patch.js

```diff
@@ -38,15 +38,15 @@ patch(Message.prototype, {
         return this.hasAuthorClickable() ? _t("Open card") : undefined;
     },
     hasAuthorClickable() {
```

Comment:
Not in this PR but I feel it could be improved too, there's no reason to limit the display of the information card to users.